go.mod

```diff
@@ -64,6 +64,8 @@ replace (
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20200331152225-585af27e34fd // release-4.5
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0 // release-4.5
 
+	github.com/operator-framework/api => github.com/joelanford/api v0.0.0-20210216180012-67617936c73e
```

Comment:
note: just need to remove this and bump the `operator-framework/api` version to a new release including the API change.